### DISCUSSION TITLE
File ownership

### DISCUSSION THREAD

**Discussion by zonta** [gitpython-developers/GitPython#1900] [Q&A] [answered]
Every time I try to use it I get this
I try to do the command said but I get an error aloso 
`fatal: detected dubious ownership in repository at 'O:/Forge/WebUi Forge/webui'
'O:/Forge/WebUi Forge/webui' is on a file system that does not record ownership
To add an exception for this directory, call:

        git config --global --add safe.directory 'O:/Forge/WebUi Forge/webui'

**Comment by Byron** [comment]
In case anyone is as puzzled as I was, this appears to be a version of [this question](https://stackoverflow.com/questions/73485958/how-to-correct-git-reporting-detected-dubious-ownership-in-repository-withou), along with the fix.

GitPython isn't mentioned, but it would certainly run into the same problem when running `git` under the hood, and what's proposed here is a fix.

**Reply by zonta** [reply, marked as answer]
OMG you solved my problem. Because there no mention to open freaking  Git Bash to input that command in and now it seems to work
